creator-node/src/services/stateMachineManager/stateReconciliation/recoverOrphanedData.jobProcessor.ts

```diff
@@ -142,67 +137,146 @@ const _storeWalletsWithThisNodeInReplica = async (
   const walletsWithNodeInReplicaSetArr: string[] = []
   do {
     // Get batch of users and all it to the array of all wallets
-    batchOfUsers = await getNodeUsers(
-      discoveryNodeEndpoint,
-      thisContentNodeEndpoint,
-      prevUserId,
-      NUM_USERS_PER_QUERY
-    )
-    batchOfUsers?.forEach((user) =>
-      walletsWithNodeInReplicaSetArr.push(user.wallet)
-    )
+    try {
+      batchOfUsers = await getNodeUsers(
```

Comment:
mind updating getNodeUsers() jsdoc from "makes single request" to e.g. "makes request with retries"